auxiliaries.py

```diff
@@ -265,10 +270,16 @@ def update(self, up_state=None):
             self.vprint(2, '  {} is UP'.format(self.hostname))
         else:
             up_state.value = 'D'
+            self.vprint(1, '  {} is DOWN!'.format(self.hostname))
+            self.vprint(1, 'Network down for {} seconds'.format(time.time() - self.last_try_time))
```

Comment:
This isn't info I want to see all the time, it clutters up the terminal display. But it would be okay on verbosity level 3.